src/utils/cookie/changedCookieHeaderValue/changedCookieHeaderValue.ts

```diff
@@ -12,7 +12,7 @@ export const changedCookieHeaderValue = (
   return [
     `${name}=${value}`,
     'path=/',
-    expires ? `Expires=${expires.toISOString()}; ` : undefined,
+    expires ? `Expires=${expires}; ` : undefined,
```

Comment:
@eunjae-lee I changes this because it did not work for me and the expires attribute was set to `Session`, now it seems to work. Please could you double-check? 

![image](https://github.com/storyblok/app-extension-auth/assets/17590227/79094d89-9d6f-4135-aeaf-a9332e1484a3)